VCNetworking/VCNetworking/operations/fetch/FetchWellKnownConfigDocumentOperation.swift

```diff
@@ -19,7 +19,15 @@ class FetchWellKnownConfigDocumentOperation: InternalNetworkOperation {
                 andCorrelationVector cv: CorrelationHeader? = nil,
                 session: URLSession = URLSession.shared) throws {
         
-        guard let baseUrl = URL(unsafeString: urlStr),
+        /// If endpoint doesn't end with / add one.
+        var endpoint: String
+        if urlStr.last != "/" {
+            endpoint = urlStr + "/"
+        } else {
+            endpoint = urlStr
+        }
+        
+        guard let baseUrl = URL(unsafeString: endpoint),
```

Comment:
Is that really the solution? What if the url is something like this:

`https://graph.microsoft.com/v1.0/users/{id}/transitiveMemberOf/microsoft.graph.group?$count=true&$orderby=displayName&$search="displayName:tier"&$select=displayName,id` (this is an example URL from graph docs)

also nit: this looks like a pretty verbose way of doing that operation. Not an expert on swift, so don't have a better way in mind and could be wrong :)

Reply:
Also this PR (also looking at the same bug in Android apparently) makes me think why it is even a problem that a trailing `/` is missing? Shouldn't this be just fine in most cases?